test/test_autograd.py

```diff
@@ -3274,21 +3274,34 @@ class dont_convert(tuple):
     ('masked_select', (), (torch.tensor(1, dtype=torch.uint8),), 'scalar'),
     ('masked_select', (M, M), (torch.tensor(1, dtype=torch.uint8),), 'scalar_broadcast_rhs'),
     ('masked_select', (), (mask_not_all_zeros((M, M)),), 'scalar_broadcast_lhs'),
+    ('masked_fill_', (M, M), (torch.ByteTensor(M, M).bernoulli_(), 10)),
```

Comment:
Here, since the semantics of `masked_fill` and `masked_fill_` are no longer the same (whether to support broadcast on `self`), it's better to test them separately.